cypress-smoketests/support/pages/local-login.js

```diff
@@ -13,6 +13,8 @@ exports.getUserToken = async username => {
 exports.login = email => {
   cy.visit(`/log-in?code=${email}`);
   cy.get('.progress-indicator').should('not.exist');
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(2000);
```

Comment:
Is there something that we should `get` that only exists once we fully loaded?  I.e. is there more stuff that loads after the `.progress-indicator` disappears that we should also wait for?

Reply:
I could try to maybe wait for something related to the work queue items to show up.  It's a bit more challenging because depending on who you log in with you'll need to wait for different page components.  I can try to find a work around for this wait.